### PR TITLE
Added support for foodfidelity.com 

### PR DESCRIPTION
**Pull request by jknndy**
This scraper includes support for the newly added grouping feature.

### CODE REVIEW

File: recipe_scrapers/foodfidelity.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+    def ratings(self):
+        return self.schema.ratings()
+
+    def cuisine(self):
+        return self.schema.cuisine()
+
+    def description(self):
+        img_tag = self.soup.find("img", {"data-pin-description": True})
+        return img_tag["data-pin-description"] if img_tag else ""
```
[the remark this answers is not in the document]
Comment (jayaddison) [reply recipe_scrapers/foodfidelity.py:55]: Oops, nope - this isn't a good suggestion.  Ignore this one, sorry.

File: recipe_scrapers/foodfidelity.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    def ingredients(self):
+        return self.schema.ingredients()
+
+    def ingredient_groups(self):
+        return group_ingredients(
+            self.ingredients(),
+            self.soup,
+            ".wprm-recipe-ingredient-group h4",
+            ".wprm-recipe-ingredient",
+        )
```
Comment (jayaddison) [inline recipe_scrapers/foodfidelity.py:42]: :100: great to see this recent functionality finding more applications!

**Comment by jayaddison** [comment]
Generally looks good to me - I'll wait to hear what you think about my suggestions @jknndy before we figure out how we proceed with these changes.

**Comment by jknndy** [comment]
I ended up implementing both changes. Had some difficulty getting the initial code block to pull the correct data from the data-tag so seeing it written in the appropriate syntax is great. 

I'll also keep in mind the return to "" instead of the hardcode for a fallback. I believe this was a similar instance to the other recently added where the author appeared to be the same across each recipe.

**Review by jayaddison — APPROVED** [review]
Thanks @jknndy - this looks good to me :+1:

**Pull request merged**